game-app/game-core/src/main/java/games/strategy/triplea/attachments/RulesAttachment.java

```diff
@@ -1057,6 +1150,12 @@ public MutableProperty<?> getPropertyOrNull(String propertyName) {
             this::setRelationship,
             this::getRelationship,
             this::resetRelationship);
+      case "checkAIPlayer":
```

Comment:
Why not a simpler name like "isAIPlayer"/

Reply:
It could be changed to "isAIPlayer", but that implies that with a value of "false" could be used to check for **not** an AI player. I use "checkAIPlayer" to infer that a value of "false" means not to check, and is the pass-through value.

Cheers...